src/core/dfn.js

```diff
@@ -18,8 +29,22 @@ export function run(conf) {
     // Also, we should probably use weakmaps and weaksets here
     // to avoid leaks.
     const $dfn = $(dfn);
-    $dfn
-      .getDfnTitles({ isDefinition: true })
+    const dfnTitles = $dfn.getDfnTitles({ isDefinition: true });
+
+    // add automatic pluralization to `data-lt` attributes
+    // see https://github.com/w3c/respec/pull/1682
+    const normText = norm(dfn.textContent);
+    if (
+      conf.pluralize === true &&
+      !dfn.hasAttribute("data-lt-noPlural") &&
+      shouldPluralize(normText)
+    ) {
+      const plural = pluralize(normText);
+      dfnTitles.push(plural);
```

Comment:
we are calling `pluralize` twice (again in `shouldPluralize()`). What about `shouldPluralize()` returns the `plural` term if valid, otherwise null otherwise so we do it with one call to `pluralize()`?

Reply:
Probably just return the empty string instead of null.